uaa/build.gradle

```diff
@@ -83,6 +83,9 @@ dependencies {
     testImplementation(libraries.springRestdocs)
     testImplementation(libraries.javaxMail)
     testImplementation(libraries.greenmail)
+
+    runtimeOnly(libraries.springDocMvc)
+    runtimeOnly(libraries.springDocSecurity)
```

Comment:
minor comment: move the order of the change, to be with other runtimeOnly configuration.

Reply:
I'm keeping these dependencies separate, to make future rebases easier.